schema/defs.json

```diff
@@ -41,10 +41,10 @@
             "minimum": 0,
             "maximum": 18446744073709552000
         },
-        "percent": {
+        "maximum": {
             "type": "integer",
             "minimum": 0,
```

Comment:
I think I agree with that, and I've made the change locally, but I'll wait for more input before I push those changes.